kubetest/e2e.go

```diff
@@ -623,10 +623,35 @@ func kubemarkTest(testArgs []string, dump, numNodes string) error {
 		})
 	}
 
+	if err := control.XmlWrap(&suite, "kubectl version", getKubectlVersion); err != nil {
+		return err
+	}
+
+	if err := control.XmlWrap(&suite, "IsUp", deploy.IsUp); err != nil {
+		return err
+	}
+
 	// Run tests on the kubemark cluster.
 	if err := control.XmlWrap(&suite, "Kubemark Test", func() error {
-		testArgs = util.SetFieldDefault(testArgs, "--ginkgo.focus", "starting\\s30\\pods")
-		return control.FinishRunning(exec.Command("./test/kubemark/run-e2e-tests.sh", testArgs...))
+		testArgs = append(
+			util.SetFieldDefault(testArgs, "--ginkgo.focus", "starting\\s30\\pods"),
+			"--e2e-verify-service-account=false",
+			"--dump-logs-on-failure=false",
```

Comment:
I think we can get rid of this (not sure why it was there before).
We're not doing it for real clusters and it should be fine to get kube-system ns logs even for kubemark I guess.

Reply:
ack